### ISSUE TITLE
Support for cancelling requests in @gravity-ui/gateway

### ISSUE THREAD

**Issue by dsleepo** [gravity-ui/rfc#25]
## Objective

There is an option In gravity-ui/sdk for cancelling requests from the browser. However, in @gravity-ui/gateway, cancelling requests from the browser is not supported, and the request from the BFF to the backend is executed until completion.

## Solution Proposal

Support API like AbortController in @gravity-ui/gateway

## Definition of done

Cancelling a request from the browser also cancels the corresponding request from the BFF to the backend.

**Comment by SeqviriouM** [comment]
In general I think it will be useful. It can be tracked on the gateway side that the client has cancelled request and also  interrupt the request on the server side. Will you be able to prepare PR for it?

**Comment by SeqviriouM** [comment]
@dsleepo can you help with PR for this ability? Or would it be better to find a volunteer?

**Comment by dsleepo** [comment]
> [@dsleepo](https://github.com/dsleepo) can you help with PR for this ability? Or would it be better to find a volunteer?

Yes, I will help. I've just created PR https://github.com/gravity-ui/gateway/pull/114

**Comment by dsleepo** [comment]
This feature was released in version 4.6.0
